.storybook/pages/Cad/Cad.tsx

```diff
@@ -10,30 +10,147 @@ import {
   LayoutContainer,
   Layout,
   LayoutSection,
-  Grid,
-  GridItem,
+  DragDrop,
+  DragDropContainerHeader,
   Icon,
-  Card,
-  CardBody,
-  OverflowList,
-  OverflowListItem,
-  Tabs,
-  Tab,
+  HorizontalStepper,
+  DataBar,
+  Toolbar,
+  ToolbarItem,
 } from '../../../src';
 
 import '../../../src/components/Utilities/Spacing.css';
-import { Toolbar } from '../../../src/upcoming-components/Toolbar/Toolbar';
-import { ToolbarItem } from '../../../src/upcoming-components/ToolbarItem/ToolbarItem';
+import ProjectCard from '../../recipes/ProjectCard';
 
-import ProjectColumn from '../../recipes/ProjectColumn';
+const EmptyImage = () => (
+  <svg
```

Comment:
could you store this `svg` in a file in `.storybook/static` (like https://github.com/chanzuckerberg/edu-design-system/pull/1073) to make this file more readable?

Reply:
Sure thing

Reply:
Done